packages/components/input/src/use-input.ts

```diff
@@ -412,7 +412,11 @@ export function useInput<T extends HTMLInputElement | HTMLTextAreaElement = HTML
         "data-focus-visible": dataAttr(isFocusVisible),
         "data-focus": dataAttr(isFocused),
         className: slots.inputWrapper({
-          class: clsx(classNames?.inputWrapper, isFilled ? "is-filled" : ""),
+          class: clsx(
+            classNames?.inputWrapper,
+            isFilled ? "is-filled" : "",
+            isMultiline ? "flex-col items-start gap-0" : "",
```

Comment:
@IsDyh01 Tailwind classes should be exclusively defined in the theme package, specifically in the `input.ts` file. This is because the TailwindCSS content path is configured to target only the `theme` package

e.g.
```
// tailwind.config.js
import {nextui} from "@nextui-org/react";

/** @type {import('tailwindcss').Config} */
const config = {
  content: [
    // ...
    // make sure it's pointing to the ROOT node_module
    "./node_modules/@nextui-org/theme/dist/**/*.{js,ts,jsx,tsx}" // <- theme package
  ],
  theme: {
    extend: {},
  },
  darkMode: "class",
  plugins: [nextui()]
}

export default config;
```